src/views/MyRequests/Table.tsx

```diff
@@ -1,22 +1,150 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
+import { useSortBy, useTable } from 'react-table';
 import { useQuery } from '@apollo/client';
-import { Link as UswdsLink } from '@trussworks/react-uswds';
+import {
+  Link as UswdsLink,
+  Table as UswdsTable
+} from '@trussworks/react-uswds';
+import classnames from 'classnames';
+import { DateTime } from 'luxon';
 import GetRequestsQuery from 'queries/GetRequestsQuery';
 import { GetRequests, GetRequestsVariables } from 'queries/types/GetRequests';
 
 import { RequestType } from 'types/graphql-global-types';
 import { formatDate } from 'utils/date';
 
 const Table = () => {
-  const { loading, error, data } = useQuery<GetRequests, GetRequestsVariables>(
-    GetRequestsQuery,
+  const { t } = useTranslation('home');
+  const { loading, error, data: tableData } = useQuery<
+    GetRequests,
+    GetRequestsVariables
+  >(GetRequestsQuery, {
+    variables: { first: 20 },
+    fetchPolicy: 'cache-and-network'
+  });
+
+  const columns: any = useMemo(() => {
+    return [
+      {
+        Header: t('requestsTable.headers.name'),
+        accessor: 'name',
+        Cell: ({ row, value }: any) => {
+          let link: string;
+          switch (row.original.type) {
+            case RequestType.ACCESSIBILITY_REQUEST:
+              link = `/508/requests/${row.original.id}`;
+              break;
+            case RequestType.GOVERNANCE_REQUEST:
+              link = `/governance-task-list/${row.original.id}`;
+              break;
+            default:
+              link = '/';
+          }
+          return (
+            <UswdsLink asCustom={Link} to={link}>
+              {value || t('requestsTable.defaultName')}
+            </UswdsLink>
+          );
+        },
+        maxWidth: 350
+      },
+      {
+        Header: t('requestsTable.headers.type'),
+        accessor: 'type',
+        Cell: ({ value }: any) => {
+          return t(`requestsTable.types.${value}`);
```

Comment:
@sarboc I'd probably create some sort of map for the type. For example

```
const requestType = {
  ACCESSIBILITY_REQUEST: t(`requestsTable.types.itGovernance`),
  GOVERNANCE_REQUEST: t(`requestsTable.types.accessibility`)
}

// Access it with
requestType[value];
```

Reply:
Ahhhhhh. Cool. I'm gonna merge this in to unblock the next PR I need to get up, but will circle back to this before the sprint is over.